src/pane-element.coffee

```diff
@@ -71,6 +71,14 @@ class PaneElement extends HTMLElement
     hasFocus = @hasFocus()
     itemView = atom.views.getView(item)
 
+    if itemPath = item.getPath?()
+      path = require 'path'
+      @dataset.activeName = path.basename(itemPath)
```

Comment:
What do you think about calling these `activeItemName` and `activeItemPath` just to be a little clearer?